solana-programs/payment-router/tests/payment-router.ts

```diff
@@ -1,105 +1,108 @@
-import * as anchor from '@coral-xyz/anchor'
-import { Program } from '@coral-xyz/anchor'
-import { PaymentRouter } from '../target/types/payment_router'
-import { SOL_AUDIO_DECIMALS, SOL_AUDIO_TOKEN_ADDRESS } from './constants'
+import * as anchor from "@coral-xyz/anchor";
+import { Program } from "@coral-xyz/anchor";
+import { PaymentRouter } from "../target/types/payment_router";
+import { SOL_AUDIO_DECIMALS, SOL_AUDIO_TOKEN_ADDRESS } from "./constants";
 import {
   TOKEN_PROGRAM_ID,
   getOrCreateAssociatedTokenAccount,
-} from '@solana/spl-token'
-import { assert } from 'chai'
+} from "@solana/spl-token";
+import { assert } from "chai";
 
-const {
-  Connection,
-  Keypair,
-  PublicKey,
-  SystemProgram,
-} = anchor.web3
+const { Connection, Keypair, PublicKey, SystemProgram } = anchor.web3;
 
 // Read in fee payer from the environment variable
-const FEE_PAYER_SECRET = process.env.feePayerSecret
+const FEE_PAYER_SECRET = process.env.feePayerSecret;
 const feePayerKeypair = Keypair.fromSecretKey(
   Uint8Array.from(JSON.parse(FEE_PAYER_SECRET))
-)
-const feePayerPublicKey = feePayerKeypair.publicKey
+);
+const feePayerPublicKey = feePayerKeypair.publicKey;
 
-const SOL_AUDIO_TOKEN_ADDRESS_KEY = new PublicKey(SOL_AUDIO_TOKEN_ADDRESS)
+const SOL_AUDIO_TOKEN_ADDRESS_KEY = new PublicKey(SOL_AUDIO_TOKEN_ADDRESS);
 
-const endpoint = 'https://api.mainnet-beta.solana.com'
-const connection = new Connection(endpoint, 'finalized')
+const endpoint = "https://api.mainnet-beta.solana.com";
+const connection = new Connection(endpoint, "finalized");
 
 async function signTransaction(transaction) {
-  transaction.partialSign(feePayerKeypair)
-  return transaction
+  transaction.partialSign(feePayerKeypair);
+  return transaction;
```

Comment:
hmmm I don't think this aligns with our prettier config fwiw 🤔

Reply:
wtf why did this happen

Reply:
vscode and lack of config in this dir i bet